live-examples/css-examples/grid/meta.json

```diff
@@ -0,0 +1,154 @@
+{
+    "pages": {
+        "gridTemplateColumns": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/grid/grid-template-columns.css",
+            "exampleCode":
+                "live-examples/css-examples/grid/grid-template-columns.html",
+            "fileName": "grid-template-columns.html",
+            "title": "CSS Demo: grid-template-columns",
+            "type": "css"
+        },
```

Comment:
It's helpful to have the entries listed alphabetically.

Reply:
done